index.html

```diff
@@ -1224,6 +1204,46 @@ <h3>Issuance Date</h3>
         </pre>
       </section>
 
+      <section>
+        <h3>Credential Subject</h3>
+
+        <p>
+A <a>verifiable credential</a> contains <a>claims</a> about a <a>subject</a>.
+The <a>subject</a> of the credential MUST be expressed using the
```

Comment:
.. claims about one or more subjects. Each subject of the credential..

Reply:
A VC can contains claims about multiple subjects, eg. a marriage certificate

Reply:
Fixed in 4f6e26dd5f2d46e12c66733463b86c5d9e5a3591.